traffic_ops/traffic_ops_golang/routing/routes.go

```diff
@@ -171,7 +171,6 @@ func Routes(d ServerData) ([]Route, []RawRoute, http.Handler, error) {
 		{1.1, http.MethodGet, `cdns/configs/?(\.json)?$`, cdn.GetConfigs, auth.PrivLevelReadOnly, Authenticated, nil, 1768437852, noPerlBypass},
 
 		{1.1, http.MethodGet, `cdns/domains/?(\.json)?$`, cdn.DomainsHandler, auth.PrivLevelReadOnly, Authenticated, nil, 296902560, noPerlBypass},
-		{1.1, http.MethodGet, `cdns/health$`, handlerToFunc(proxyHandler), 0, NoAuth, []middleware.Middleware{}, 1443074329, noPerlBypass},
```

Comment:
Oy, the route ordering strikes again. I think this route was here because it otherwise gets eaten by the less specific `GET /cdns/{id}` route. So the new `GET /cdns/health` routes need to be moved up here as well.

Reply:
Ah, I'll fix. Give me a bit, currently using my machine to diff configs, planning to test this when that's done.

Reply:
Fixed